src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -202,39 +208,45 @@
-            first-route                  (first route)
-            native-currency-symbol       (get-in first-route [:from :native-currency-symbol])
-            native-token                 (when native-currency-symbol
-                                           (rf/sub [:wallet/token-by-symbol native-currency-symbol]))
-            fee-formatted                (rf/sub [:wallet/wallet-send-fee-fiat-formatted native-token])
-            account                      (rf/sub [:wallet/current-viewing-account])
-            account-color                (:color account)
-            bridge-to-network            (when bridge-to-chain-id
-                                           (rf/sub [:wallet/network-details-by-chain-id
-                                                    bridge-to-chain-id]))
-            loading-suggested-routes?    (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
-            from-account-props           {:customization-color account-color
-                                          :size                32
-                                          :emoji               (:emoji account)
-                                          :type                :default
-                                          :name                (:name account)
-                                          :address             (utils/get-shortened-address (:address
-                                                                                             account))}
-            user-props                   {:full-name to-address
-                                          :address   (utils/get-shortened-address to-address)}]
+                    to-values-by-chain type recipient]} send-transaction-data
+            collectible?                                (some? collectible)
+            image-url                                   (when collectible
+                                                          (get-in collectible [:preview-url :uri]))
+            transaction-type                            (:tx-type send-transaction-data)
+            estimated-time-min                          (reduce + (map :estimated-time route))
+            token-symbol                                (or token-display-name
+                                                            (-> send-transaction-data :token :symbol))
+            first-route                                 (first route)
+            native-currency-symbol                      (get-in first-route
+                                                                [:from :native-currency-symbol])
+            native-token                                (when native-currency-symbol
+                                                          (rf/sub [:wallet/token-by-symbol
+                                                                   native-currency-symbol]))
+            fee-formatted                               (rf/sub [:wallet/wallet-send-fee-fiat-formatted
```

Comment:
Sometimes it's a smell when there are too many let bindings. This `view` component is one such example. Here, for example, `fee-formatted` is only used by the `transaction-details` component. Also, the let bindings `first-route`, `native-currency-symbol`, `native-token` are all used only to compute the value of `fee-formatted`.

Also `estimated-time-min` can be calculated inside `transaction-details`. I stopped checking here ;)

Therefore, all these temporary let bindings that make reading the component more laborious can be moved to a separate subscription, and which can be easily unit tested.

The other important bit is that subscriptions should be used as close as possible to the actual component using them, otherwise global state changes become more expensive because more components in the tree need to be reprocessed. This requires our constant attention while implementing because it's expensive for reviewers to catch these problems (usually requiring multiple searches in the file/project to assess the situation, which I had to do now).

If this is too much to handle in this PR, it'd be good to keep a closer look in the future and avoid in the first place because it's a debt we don't have to create.